sirepo/server.py

```diff
@@ -85,7 +85,9 @@
 
 def api_blueskyAuth():
     from sirepo import bluesky
-    return _json_response(bluesky.auth_login(_json_input()))
+    req = _json_input()
+    sim_type = req['simulationType']
+    return _json_response(bluesky.auth_login(req, schema=simulation_db.get_schema(sim_type)))
```

Comment:
It would be better to move the response from bluesky (lines 104-108), and just do this in server.py:

```python
req = _json_input()
bluesky.auth_login(req)
return _json_response(
    status='OK',
    data=simulation_db.open_json_file(req.simulationType, sid=req.simulationId),
    schema=simulation_db.get_schema(req.simulationType)),
)
```

This would be less coupling.

Why is "status='OK'" used when the rest of sirepo use "state='ok'"?

Reply:
Good idea, @robnagler! I'll update the code and test it.

Not sure about `status='OK'`, it was originally there even before your update https://github.com/radiasoft/sirepo/commit/85dfe66742ae14e92ea526a3b4304bce834387be. @moellep was the first who implemented that in https://github.com/mrakitin/sirepo/commit/267fa6f87dd0118e4de475a044b5b4f569cacd4e. Should it be changed to `ok`?

Reply:
Seems like we should share a common "ok". In one case it is state and another it is status. I'm sure it was just an oversight.

I think we should maybe add and argument to _json_response_ok() so that it can merge the state so we don't have to repeat "state='ok'" in the code.

Reply:
I'm happy with whatever solution you propose regarding `OK/ok`. I'm not sure it should be a part of this PR though.

Reply:
Btw, should it be `req.simulationType` or `req['simulationType']`? I remember there were some disagreements what we should use.